lib/workers/pr/index.ts

```diff
@@ -343,15 +344,15 @@ export async function ensurePr(
       }
       existingPrBody = existingPrBody.trim();
       if (
-        existingPr.title === prTitle &&
-        noWhitespaceOrHeadings(existingPrBody) ===
-          noWhitespaceOrHeadings(prBody)
+        stripEmojis(existingPr.title) === stripEmojis(prTitle) &&
+        noWhitespaceOrHeadings(stripEmojis(existingPrBody)) ===
+          noWhitespaceOrHeadings(stripEmojis(prBody))
       ) {
         logger.debug(`${existingPr.displayNumber} does not need updating`);
         return { prResult: PrResult.NotUpdated, pr: existingPr };
       }
       // PR must need updating
-      if (existingPr.title !== prTitle) {
+      if (stripEmojis(existingPr.title) !== stripEmojis(prTitle)) {
```

Comment:
We should store `stripEmojis(existingPr.title)` and `stripEmojis(prTitle)` to a variable and reuse (perf, mem)